zerver/openapi/zulip.yaml

```diff
@@ -524,6 +525,7 @@ paths:
           The number of messages with IDs less than the anchor to retrieve.
         schema:
           type: integer
+          minimum: 0
```

Comment:
Can you please explain what error you got without this change?  I'm not sure that this is correct and I'd also like to understand how the validation works.

Reply:
In the function `check_bad_int_type` (which I have changed in the last commit) there is a comment:
```
        """
        num_before, num_after, and narrow must all be non-negative
        integers or strings that can be converted to non-negative integers.
        """
```
and that function provides negative values to num_before, num_after and expects json error. So I reflected this non-negative property in zulip.yaml by setting `minimum` value fo these parameters to 0.